frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -0,0 +1,379 @@
+            id="select-region"
+            placeholder="Toutes les régions"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4">
+          <label for="select-department" class="text-body-2">
+            Département
+          </label>
+          <v-autocomplete
+            v-model="chosenDepartment"
+            :items="departments"
+            clearable
+            hide-details
+            id="select-department"
+            placeholder="Tous les départements"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4" class="d-flex align-end">
+          <v-btn x-large color="primary" @click="submit">
+            <!-- More descriptive text? -->
+            Rechercher
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+    <div v-if="locationText" class="py-8">
+      <h2 class="text--darken-5 text-h4 mb-8">Les statistiques pour {{ locationText }}</h2>
+      <v-row :class="{ 'flex-column': $vuetify.breakpoint.smAndDown }">
+        <v-col cols="12" md="6" class="pr-0">
+          <div id="published-canteen-text" class="mb-5">
+            <p>
+              Aujourd'hui, il y a
+              <span class="text-h5 font-weight-bold">{{ statistics.canteenCount }}</span>
+              cantine{{ statistics.canteenCount == 1 ? "" : "s" }} dans {{ locationText }} sur ce site.
+            </p>
+            <p>
+              <span class="text-h5 font-weight-bold">{{ statistics.publishedCanteenCount }}</span>
+              cantine{{ statistics.publishedCanteenCount == 1 ? " a publié ses" : "s ont publié leurs" }} données,
+              accessible par
+              <router-link :to="{ name: 'CanteensHome' }">nos cantines</router-link>
+            </p>
+          </div>
+          <VueApexCharts
+            :options="publishedChartOptions"
+            :series="publishedSeries"
+            type="pie"
+            height="auto"
+            v-if="$vuetify.breakpoint.mdAndUp"
+            width="62%"
+            role="img"
+            aria-describedby="published-canteen-text"
+          />
+        </v-col>
+        <v-col cols="12" sm="8" md="6" class="pl-0">
+          <VueApexCharts
+            :options="sectorChartOptions"
+            :series="sectorSeries"
+            type="bar"
+            height="auto"
+            width="100%"
+            role="img"
+            :aria-label="sectorChartTitle"
+            aria-describedby="sector-chart-description"
+          />
+          <p id="sector-chart-description" class="d-none">{{ sectorChartDescription }}</p>
+        </v-col>
+      </v-row>
+      <h3 class="text-h5 mt-10 mb-8 text--darken-5">Qualité de produits en {{ year }}</h3>
```

Comment:
Dans la même ligne que mon commentaire au-dessus par rapport à l'épaisseur des titres, ce niveau pourrait être un truc comme : 
```suggestion
      <h3 class="text-h6 font-weight-black mt-10 mb-8">Qualité de produits en {{ year }}</h3>
```